packages/hop-node/src/decorators/rateLimitRetry.ts

```diff
@@ -47,7 +47,7 @@ export function rateLimitRetryFn (fn: any): any {
         if (retries >= rateLimitMaxRetries) {
           logger.error(`max retries (${rateLimitMaxRetries}) reached. Error: ${err.message}`)
```

Comment:
suggestion: log the entire `err` here to show stack trace

```
logger.error(`max retries (${rateLimitMaxRetries}) reached. Error:`, err)
```

Reply:
Resolved with e418f459dd8a5b729468eb198b290f71dc623d1f